src/main/java/com/stripe/net/ApiResource.java

```diff
@@ -211,6 +211,12 @@ public static InputStream requestStream(
     return ApiResource.stripeResponseGetter.requestStream(method, url, params, options);
   }
 
+  public static StripeResponse rawRequestStream(
```

Comment:
Do we need the stream version of raw request?

Reply:
Ah you're right, we don't need the stream version. Updated